utopian-io/v2.utopian.io
# Bugfix/auth signup redirect

### Verify that

### My PR doesn't concern a Jira task
Fixes in the auth app:
- Add uid in the access token after account creation.
- Add missing i18n words
- Fix redirect when there's no redirectUrl set
---

### **Test plan**

Redirect issue:
* Go directly to localhost:8081 and sign in. You should be redirected to the main app if you have an account.

Account creation access token:
* Create a new user
* Verify that the access token returned and set in the cookies contains the `uid`
